src/main/java/core/basesyntax/service/impl/ReportServiceCsvImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceCsvImpl implements ReportService {
+    private static final String FIRST_LINE = "fruit,quantity\n";
```

Comment:
The header line 'fruit,quantity' is hardcoded. If the CSV format changes or additional fields are needed, you will need to modify this line. Consider defining the header line closer to where the file's fields are determined, or make it configurable if the CSV format may vary.